[Observability AI Assistant] Add shared conversations (#4000)
elastic/docs-content: solutions/observability/observability-ai-assistant.md

`get_apm_services_list`
:   Get the list of monitored services, their health statuses, and alerts.

#### Share conversations

> Conversations that you start with AI Assistant are private by default and not visible to other members of your team. Sharing conversations can help you keep your teammates informed about your conversations with AI Assistant, making it easier to simultaneously collaborate with your teammates and AI Assistant.

Reviewer: > not visible to other members of your team.

Should we be more explicit that a shared conversation is visible to everyone in the space?
I'm not sure whether the word "team" implies that.

To share a conversation:

1. Select the **Private** / **Shared** badge next to the conversation's title
1. Use the dropdown menu to define the chat's visibility.

:::{image} /solutions/images/obs-ai-assistant-shared-conversations.png
:alt: AI Assistant chat with the sharing status dropdown open
:screenshot:
:::

After sharing a conversation, you can copy its URL and send the link to your team:

1. Open an AI Assistant chat.
1. Open the **Conversation actions** ({icon}`boxes_vertical`) menu.
1. Select **Copy URL**.

When someone shares a chat with you, you can review it, but you can't continue the conversation. To continue the conversation where your colleague left off, duplicate the conversation.

To duplicate a shared conversation:

1. Open an AI Assistant chat.
1. Open the **Conversation actions** ({icon}`boxes_vertical`) menu.
1. Select **Duplicate**.

#### Archive conversations

> The owner of a conversation can archive it by selecting **Archive** from the **Conversation actions** ({icon}`boxes_vertical`) menu. Users can't continue or edit archived conversations.

Reviewer 2: Based on the next sentence, it feels more accurate to say:

```suggestion
The owner of a conversation can archive it by selecting **Archive** from the **Conversation actions** ({icon}`boxes_vertical`) menu. Once archived, a conversation can't be edited or continued until it is unarchived.
```

`get_apm_services_list`
:   Get the list of monitored services, their health statuses, and alerts.

#### Share conversations

Conversations that you start with AI Assistant are private by default and not visible to other users in the space. Sharing conversations lets everyone in the space see what you've already asked or learned, making it easier to collaborate.

To change the visibility of a conversation:

1. Select the **Private** / **Shared** badge next to the conversation's title
1. Use the dropdown menu to define the chat's visibility.

:::{image} /solutions/images/obs-ai-assistant-shared-conversations.png
:alt: AI Assistant chat with the sharing status dropdown open
:screenshot:
:::

After sharing a conversation, you can copy its URL and send the link to your team:

1. Open an AI Assistant chat.
1. Open the **Conversation actions** ({icon}`boxes_vertical`) menu.
1. Select **Copy URL**.

> When someone shares a chat with you, you can review it, but you can't continue the conversation. To continue the conversation where your colleague left off, duplicate the conversation.

Reviewer: ```suggestion
When someone shares a chat with you, you can view it, but you can't edit or continue the conversation. To continue the conversation where your colleague left off, duplicate the conversation.
```
